pkg/sql/logictest/testdata/logic_test/distsql_enum

```diff
@@ -70,20 +70,9 @@ EXPLAIN (VEC)
 SELECT x from t1 WHERE EXISTS (SELECT x FROM t2 WHERE t1.x=t2.x AND t2.y='hello')
 ----
 │
-├ Node 1
```

Comment:
do you know why this changed?

Reply:
Fixed, I had a bug by not performing the cast. I plumbed the cast though.